trashapplet/src/trashapplet.c

```diff
@@ -128,12 +131,15 @@ trash_applet_monitor_changed (TrashApplet *applet)
                                             "%d Items in Trash",
                                             items), items);
           gtk_widget_set_tooltip_text (GTK_WIDGET (applet), text);
+          atk_object_set_description (atk_obj, text);
           g_free (text);
         }
-      else
-        gtk_widget_set_tooltip_text (GTK_WIDGET (applet),
-                                     _("No Items in Trash"));
-
+      else {
+        char *text;
```

Comment:
const char *text =  _("No Items in Trash");

Reply:
I’m not sure I understand what you mean; something like that?
```
  if (items != applet->items)
    {
      const char *text = _("No Items in Trash");
      if (items)
        {
          text = g_strdup_printf (ngettext ("%d Item in Trash",
                                            "%d Items in Trash",
                                            items), items);
        }
      gtk_widget_set_tooltip_text (GTK_WIDGET (applet), text);
      g_free (text);

      applet->items = items;
    }
```

Reply:
```
const char *text =  _("No Items in Trash");
gtk_widget_set_tooltip_text (GTK_WIDGET (applet), text);
atk_object_set_description (atk_obj, text);
```

Reply:
I don't think it's necessary to allocate memory for string constants at all